tests/framework/i18n/ExposedI18N.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace yiiunit\framework\i18n;
+
+use yii\i18n\I18N;
+
+/**
+ * ExposedI18N is a version of I18N component with
+ * protected methods exposed for testing purposes
+ *
+ * @package yiiunit\framework\i18n
+ */
+class ExposedI18N extends I18N
+{
+    /**
+     * @inheritdoc
+     */
+    public function fallbackNormalizeLocale($locale)
+    {
+        return parent::fallbackNormalizeLocale($locale);
```

Comment:
See for example:
https://github.com/yiisoft/yii2-bootstrap/pull/31/files#diff-2f1760bba264cf4f4b0167dbbe1599e3R21